src/dropdown/test/dropdown.spec.js

```diff
@@ -587,6 +587,17 @@ describe('dropdownToggle', function() {
       expect(isFocused(focusEl)).toBe(false);
     });
 
+    it('should focus last list element when up arrow pressed after dropdown toggled', function() {
+      $document.find('body').append(element);
+      clickDropdownToggle();
+      expect(element.hasClass(dropdownConfig.openClass)).toBe(true);
+
+      triggerKeyDown($document, 38);
+      var elems = element.find('ul').eq(0).find('a');
```

Comment:
the selector isn't the problem, rather the up trigger is not causing the anchor tag to become the `document.activeElement`. so the focus check will fail.  why this is the case, i don't know.  but essentially, the up arrow key is doing nothing here.